pom.xml

```diff
@@ -35,6 +35,18 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-module-junit4</artifactId>
+            <version>1.6.5</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.diffblue</groupId>
+            <artifactId>deeptestutils</artifactId>
```

Comment:
I would also get rid of this and the `staticInitOutputVoid` test. Let's assume in this PR that we are using test-ranker and no-trivial test option which will remove that test.